protobuf/drspec/review_record.proto

```diff
@@ -0,0 +1,154 @@
+syntax = "proto2";
+
+import "popr.proto";
+import "signature.proto";
+import "rating.proto";
+
+message ReviewRecord {
+  
+  message DetailedReview {
+
+    required string attribute = 1;
+
+    required string review_text = 2;
+
+    required Rating rating = 3;
+
+    optional string category = 4;
+
+  }
+
+  // identify the blockchain, should btc, tbtc, eth, etc
+  required string currency_symbol = 1;
+  
+  required int32 amount = 2;
+
+  required string customer_address = 3;
+
+  required string vendor_address = 4;
+
+  // goes inside a new review field
+  optional string OBSOLETE_review_text = 5;
+
+  // goes inside a new review field
+  optional uint32 OBSOLETE_rating = 6;
+  
+  repeated DetailedReview detailed_review = 7;
+
+  required PoPR popr = 8;
+
+  required uint32 chlu_version = 9;
+
+  optional string last_reviewrecord_multihash = 11;
+
+  required string hash = 12;
+
+  // Shouldn't be used anymore
+  required string signature = 13;
+
+  optional string key_location = 14;
+
+  optional string previous_version_multihash = 15;
+
+  // goes into the issuer field
+  optional string OBSOLETE_customer_did_id = 16;
+
+
+  // Supporting spec
+  required string id = 17;
+
+  message Location {
+
+    required float lat = 1;
+
+    required float lng = 2;
+  }
+
+  message Subject {
+
+    required string did = 1;
+
+    optional string name = 2;
+
+    optional string address = 3;
+
+    optional string telephone = 4;
+
+    repeated string categories = 5;
+
+    optional Location location = 6;
+
+    optional string url = 7;
+  }
+
+  message Platform {
+
+    required string name = 1;
+
+    required string url = 2;
+
+    optional string subject_url = 3;
+  }
+
+  message Author {
+
+    required string name = 1;
+
+    optional string platform_url = 2;
```

Comment:
we have a `platform` with a `url` and `subject_url`, is this `author.platform_url` a different URL or is it redundant?

Reply:
It is the URL of the customer profile on the marketplace, if one exists.